crates/ruma-state-res/src/lib.rs

```diff
@@ -229,7 +234,7 @@ fn reverse_topological_power_sort<E: Event>(
     let mut event_to_pl = HashMap::new();
     for event_id in graph.keys() {
         let pl = get_power_level_for_sender(event_id.borrow(), &fetch_event)?;
-        info!("{event_id} power level {pl}");
+        debug!(%event_id, power_level = %pl, "found the power level of an event's sender");
```

Comment:
```suggestion
        debug!(?event_id, power_level = %pl, "found the power level of an event's sender");
```

I'd prefer to be consistent with the SDK and use debug formatting for string-like types (which means that quotes are included).

Reply:
Hmm, that seems kinda odd. When writing events to a structured format like JSON, these values would be double-quoted if we made this change. This would make searching things in trace aggregators more confusing IMO, especially since I think it's likely that people will default to Display impls over Debug impls if they exist in traces in the rest of their application.

Reply:
That's unfortunate, but so is _not_ including the quotes when printing to stdout / stderr. It's pretty verbose, but would you mind doing this then?

```suggestion
        debug!(event_id = event_id.as_str(), power_level = %pl, "found the power level of an event's sender");
```

Long-term, tracing wants to stabilize support for custom types for fields, which would mean simply implementing a trait for our ID types, but it's not yet possible (seems like there has at least been a little progress on it this year in https://github.com/tokio-rs/tracing/pull/2960).

Reply:
> That's unfortunate, but so is not including the quotes when printing to stdout / stderr.

Yeah I don't disagree.

> It's pretty verbose, but would you mind doing this then?

Sure.

Reply:
Fixed these.